src/SprykerEco/Zed/UnzerRestApi/Business/UnzerRestApiFacadeInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerRestApi\Business;
+
+use Generated\Shared\Transfer\RestCheckoutDataTransfer;
+
+interface UnzerRestApiFacadeInterface
+{
+    /**
+     * Specification:
+     * - Requires `QuoteTransfer.store.name` transfer property to be set.
+     * - Expands `QuoteTransfer` with `UnzerCredentialsTransfer` according to added items.
+     * - Does nothing if quote has no items.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\RestCheckoutDataTransfer $restCheckoutDataTransfer
+     *
+     * @return \Generated\Shared\Transfer\RestCheckoutDataTransfer
+     */
+    public function expandCheckoutData(RestCheckoutDataTransfer $restCheckoutDataTransfer): RestCheckoutDataTransfer;
```

Comment:
* The test must be written to cover the new facade method.

Reply:
Basically, I agree with you, but the situation is a little... weird. Can you please say something about that @kkicheglovspryker? :)

Reply:
Fixed.